sdk/src/test/java/com/hedera/hashgraph/sdk/TokenDissociateTransactionTest.java

```diff
@@ -77,11 +76,46 @@ void shouldBytes() throws Exception {
     @Test
     void fromScheduledTransaction() {
         var transactionBody = SchedulableTransactionBody.newBuilder()
-            .setTokenDissociate(TokenDissociateTransactionBody.newBuilder().build())
-            .build();
+            .setTokenDissociate(TokenDissociateTransactionBody.newBuilder().build()).build();
 
         var tx = Transaction.fromScheduledTransaction(transactionBody);
 
         assertThat(tx).isInstanceOf(TokenDissociateTransaction.class);
     }
+
+    @Test
+    void constructTokenMintTransactionFromTransactionBodyProtobuf() {
```

Comment:
I think the name is wrong here it should be about token dissociate not mint

Reply:
@rokn updated this one.